---
layout: page
title: Science
subtitle: Research and scientific explorations
---



## Bacterial Adhesins and Autotransporters

*   **Adhesion of the genome-sequenced Lactococcus lactis subsp. cremoris IBB477 strain is mediated by specific molecular determinants**. Radziwill-Bienkowska JM, Le DTL, **Szczesny P**, Duviau MP, Aleksandrzak-Piekarczyk T, Loubière P, Mercier-Bonin M, Bardowski JK, Kowalczyk M. *Applied Microbiology and Biotechnology*. 2016. 100:9605-9617.
*   **Structure, domain diversity and evolution of trimeric autotransporter adhesins**. **Szczesny P**, Lupas A, Linke D. In: *61. Jahrestagung der Deutschen Gesellschaft fur Hygiene und Mikrobiologie (DGHM 2009)*. 2009. p. 64. Urban & Fischer.
*   **Domain annotation of trimeric autotransporter adhesins—daTAA**. **Szczesny P**, Lupas A. *Bioinformatics*. 2008. 24(10):1251-1256.
*   **Structure of the head of the Bartonella adhesin BadA**. **Szczesny P**, Linke D, Ursinus A, Bär K, Schwarz H, Riess TM, Kempf VAJ, Lupas AN, Martin J, Zeth K. *PLoS Pathogens*. 2008. 4(8):e1000119.
*   **A conserved glycine residue of trimeric autotransporter domains plays a key role in Yersinia adhesin A autotransport**. Grosskinsky U, Schütz M, Fritz M, Schmid Y, Lamparter MC, **Szczesny P**, Lupas AN, Autenrieth IB, Linke D. *Journal of bacteriology*. 2007. 189(24):9011-9019.
*   **Model structure of the prototypical non-fimbrial adhesin YadA of Yersinia enterocolitica**. Koretke KK, **Szczesny P**, Gruber M, Lupas AN. *Journal of structural biology*. 2006. 155(2):154-161.

## Genomics, Metagenomics, and Bioinformatics Tools

*   **Inferring gene-species assignments in the presence of horizontal gene transfer**. Mykowiecka A, **Szczesny P**, Górecki P. *IEEE/ACM transactions on computational biology and bioinformatics*. 2017. 15(5):1571-1578.
*   **Detection of methanogens and phylogenetic analyses of methanogenic communities**. **Szczesny P**, Lipinski L, Bartosik D, Drewniak L. In: *From Genes to Species: Novel Insights from Metagenomics*. 2016. p. 92. Frontiers Media SA.
*   **Extending MetAMOS-new methods and new integrations**. Siwiak M, Bogdanowicz A, Hajduk A, Krassowski M, Jankowski P, Savenko M, Pyzik AAP, **Szczesny P**. *PeerJ PrePrints*. 2016. 4:e1706v1.
*   **Novel molecular markers for the detection of methanogens and phylogenetic analyses of methanogenic communities**. Dziewit L, Pyzik A, Romaniuk K, Sobczak A, **Szczesny P**, Lipinski L, Bartosik D, Drewniak L. *Frontiers in Microbiology*. 2015. 6:694.
*   **Fast algorithms for inferring gene-species associations**. Betkier A, **Szczesny P**, Górecki P. In: *International Symposium on Bioinformatics Research and Applications*. 2015. pp. 36-47. Springer International Publishing Cham.
*   **Algorithm for error detection in metagonomics NGS data**. Vassilev D, Krachunov M, Popov I, Todorovska E, Simeonova V, **Szczesny P**, Siedlecki P, Zielenkiewicz U. *EMBnet. journal*. 2012. 17(B):pp-28.
*   **MOFOID—not only the protein modeling server**. **Szczesny P**, Wieczorek G, Zielenkiewicz P. *Acta Biochim. Pol*. 2005. 52:267-269.

## Protein Evolution, Phylogenetics, and Horizontal Gene Transfer

*   **Distinct Protein Classes in Human Red Cell Proteome Revealed by Similarity of Phylogenetic Profiles**. **Szczesny P**, Mykowiecka A, Pawłowski K, Grynberg M. *PloS one*. 2013. 8(1):e54471.
*   **The evolutionary rate of antibacterial drug targets**. Gladki A, Kaczanowski S, **Szczesny P**, Zielenkiewicz P. *BMC bioinformatics*. 2013. 14:1-10.
*   **Recurrent horizontal transfer of bacterial toxin genes to eukaryotes**. Moran Y, Fredman D, **Szczesny P**, Grynberg M, Technau U. *Molecular Biology and Evolution*. 2012.
*   **Extending the Aerolysin Family: From Bacteria to Vertebrates**. **Szczesny P**, Iacovache I, Muszewska A, Ginalski K, van der Goot FG, Grynberg M. *PloS one*. 2011. 6(6):e20349.
*   **Independent Subtilases Expansions in Fungi Associated With Animals**. Muszewska A, Taylor JW, **Szczęsny P**, Grynberg M. *Molecular Biology and Evolution*. 2011.

## Translational Regulation and RNA Biology

*   **Plasmodium falciparum translational machinery condones polyadenosine repeats**. Pavlovic Djuranovic S, Erath J, Andrews RJ, Bayguinov PO, Chung JJ, Chalker DL, Fitzpatrick JAJ, Moss WN, **Szczesny P**, Djuranovic S. *Elife*. 2020. 9:e57799.
*   **A short translational ramp determines the efficiency of protein synthesis**. Verma M, Choi J, Cottrell KA, Lavagnino Z, Thomas EN, Pavlovic-Djuranovic S, **Szczesny P**, Piston DW, Zaher HS, Puglisi JD, and others. *Nature communications*. 2019. 10(1):5774.
*   **PolyA tracks and poly-lysine repeats are the Achilles heel ofPlasmodium falciparum**. Djuranovic SP, Erath J, Andrews RJ, Bayguinov PO, Chung JJ, Chalker DL, Fitzpatrick JA, Moss WN, **Szczesny P**, Djuranovic S. 2018.
*   **Translation efficiency is a determinant of the magnitude of miRNA-mediated repression**. Cottrell KA, **Szczesny P**, Djuranovic S. *Scientific reports*. 2017. 7(1):14884.
*   **Two novel C-terminal frameshift mutations in the $\beta$-globin gene lead to rapid mRNA decay**. Rawa K, Szczesny RJ, Owczarek EP, Adamowicz-Salach A, Klukowska A, Demkow U, Plochocka D, **Szczesny P**, Gora M, Dziembowski A, and others. *BMC Medical Genetics*. 2017. 18(1):65.
*   **PATACSDB-The database of polyA translational attenuators in coding sequences**. Habich M, Djuranovic S, **Szczesny P**. *PeerJ Computer Science*. 2016. 2:e45.
*   **Synonymous mutations in the light of translational regulation**. **Szczesny P**. In: *BIO 2016 Congress*. 2016. Vol. 2, p. 91. Acta Biochimica Polonica.
*   **Translational control by lysine-encoding A-rich sequences**. Arthur LL, Pavlovic-Djuranovic S, Koutmou KS, Green R, **Szczesny P**, Djuranovic S. *Science advances*. 2015. 1(6):e1500154.

## Microbial Ecology and Biotechnology

*   **Evaluation of bacterial diversity on historical silk velvet textiles from the Museum of King John III's Palace at Wilanow, Poland**. Brzozowska I, Bogdanowicz A, **Szczesny P**, Zielenkiewicz U, Laudy A. *International Biodeterioration & Biodegradation*. 2017.
*   **Taxonomic and chemical assessment of exceptionally abundant rock mine biofilm**. Tomczyk-Żak K, **Szczesny P**, Gromadka R, Zielenkiewicz U. *PeerJ*. 2017. 5:e3635.
*   **Noteworthy Facts about a Methane-Producing Microbial Community Processing Acidic Effluent from Sugar Beet Molasses Fermentation.** Chojnacka A, **Szczesny P**, Błaszczyk MK, Zielenkiewicz U, Detman A, Salamon A, Sikora A. *PLoS One*. 2015. 10(5):e0128008-e0128008.
*   **Uranium bioleaching-insight into the structure of microbial consortia from mining tailings**. Zielenkiewicz U, **Szczesny P**. *The Febs Journal*. 2013. 280(1):570.
*   **Biodiversity in Oscypek, a traditional Polish cheese, determined by culture-dependent and-independent approaches**. Alegría Á, **Szczesny P**, Mayo B, Bardowski J, Kowalczyk M. *Applied and environmental microbiology*. 2012. 78(6):1890-1898.
*   **Comparative analysis of hydrogen-producing bacterial biofilms and granular sludge formed in continuous cultures of fermentative bacteria**. Chojnacka A, Blaszczyk MK, **Szczesny P**, Nowak K, Suminska M, Tomczyk-Zak K, Zielenkiewicz U, Sikora A. *Bioresource Technology*. 2011.

## Physiology, Metabolism, and Medical Research

*   **Correlation of gestational age and age at death in sudden infant death syndrome: another pointer to the role of critical developmental period?**. Habich M, Zielenkiewicz P, Paczek L, **Szczesny P**. *BMC pediatrics*. 2024. 24(1):259.
*   **The relationship between EMG high frequency and low frequency band amplitude changes correlates with tissue inorganic phosphate levels**. Habich M, Pawlinski B, Lorenc K, Sady M, Siewruk K, Zielenkiewicz P, Gajewski Z, Poznanski J, Paczek L, **Szczesny P**. *Acta Biochimica Polonica*. 2023.
*   **Stress-induced phosphaturia in weaned piglets**. Habich M, Pawlinski B, Sady M, Siewruk K, Zielenkiewicz P, Gajewski Z, **Szczesny P**. *Animals*. 2020. 10(12):2220.
*   **Combined in silico and 19F NMR analysis of 5-fluorouracil metabolism in yeast at low ATP conditions**. Pawłowski PH, **Szczesny P**, Rempoła B, Poznańska A, Poznański J. *Bioscience Reports*. 2019. 39(12):BSR20192847.
*   **Correlation between basic biochemical and gasometric parameters and distribution of follicle size in Holstein Friesians (HF) cows**. Pawlinski B, Trela M, Sady M, Gajewska M, Domino M, **Szczesny P**, Gajewski Z. In: *REPRODUCTION IN DOMESTIC ANIMALS*. 2018. Vol. 53, pp. 30-31. 
*   **Arteriovenous oscillations of the redox potential: Is the redox state influencing blood flow?**. Poznanski J, **Szczesny P**, Pawlinski B, Mazurek T, Zielenkiewicz P, Gajewski Z, Paczek L. *Redox Report*. 2017. 22(5):210-217.
*   **Evaluation of selected gasometric parameters in piglets during weaning period**. Pawlinski B, Pawlinski B, **Szczesny P**, Siewruk K, Domino M, Sady M, Gajewska M, Gajewski Z. In: *REPRODUCTION IN DOMESTIC ANIMALS*. 2017. Vol. 52, pp. 121-122. WILEY 111 RIVER ST, HOBOKEN 07030-5774, NJ USA.
*   **Hypophosphatemia and sudden infant death syndrome (SIDS)—is ATP the link?**. **Szczesny P**, Poznanski J, Paczek L, Zielenkiewicz P. *Upsala journal of medical sciences*. 2014. 119(1):55-56.
*   **Proteins contribute insignificantly to the intrinsic buffering capacity of yeast cytoplasm**. Poznanski J, **Szczesny P**, Ruszczyńska K, Zielenkiewicz P, Paczek L. *Biochemical and biophysical research communications*. 2013. 430(2):741-744.
*   **Fluorouracil metabolism in Saccharomyces cerevisiase by 19F NMR**. Rempola B, Krupinski P, Polkowska A, **Szczesny P**, Pawlowski P, Poznanski J. In: *YEAST*. 2013. Vol. 30, pp. 171-171. 

## Open Science and Scientific Communication

*   **Foundations for open scholarship strategy development**. Tennant J, Beamer J, Bosman J, Brembs B, Chung NC, Clement G, Crick T, Dugan J, Dunning A, Eccles D, and others. *MetaArXiv*. 2019. 
*   **Otwarta nauka, czyli dobre praktyki uczonych (Open science - good research practices)**. **Szczęsny P**. *Wydawnictwa Zwarte Stowarzyszenia EBIB*. 2013. Vol. 7. Wydawnictwa Zwarte Stowarzyszenia EBIB.
*   **Otwieranie nauki -- lekcja z globalizacji (Opening science - the lesson from globalisation)**. **Szczesny P**. *Biuletyn Polskiego Towarzystwa Ekonomicznego*. 2012. 56:138-143.
*   **Nauka 2.0. Świadome współtworzenie**. **Szczęsny P**. Instytut Obywatelski, 2012.
*   **Acknowledging contributions to online expert assistance**. Waagmeester A, Palidwor G, **Szczesny P**, Albert I, Mangan M, Miller C, Cockell S, Lindenbaum P, Silvestre DAMM, Dall’Olio GM, and others. Altmetrics Workshop. 2011.

## Plant Genomics and Physiology

*   **Engineering high Zn in tomato shoots through expression of AtHMA4 involves tissue-specific modification of endogenous genes**. Kendziorek M, Klimecka M, Barabasz A, Borg S, Rudzka J, **Szczesny P**, Antosiewicz DM. *BMC genomics*. 2016. 17:1-20.
*   **Potato annexin STANN1 promotes drought tolerance and mitigates light stress in transgenic Solanum tuberosum L. plants**. Szalonek M, Sierpien B, Rymaszewski W, Gieczewska K, Garstka M, Lichocka M, Sass L, Paul K, Vass I, Vankova R, and others. *PloS one*. 2015. 10(7):e0132683.
*   **Genome sequence and analysis of the tuber crop potato.** Xu X, Pan S, Cheng S, Zhang B, Mu D, Ni P, Zhang G, Yang S, Li R, Wang J, and others. *Nature*. 2011. 475(7355):189-195.
*   **Genom ziemniaka zsekwencjonowany**. Gromadka R, Gawor J, **Szczęsny P**, Zagórski W. *KOSMOS*. 2011. 60(3-4):491-497.

## Others

*   **From the index case to global spread: the global mobility based modelling of the COVID-19 pandemic implies higher infection rate and lower detection ratio than current estimates**. Siwiak M, **Szczesny P**, Siwiak M. *PeerJ*. 2020. 8:e9548.
*   **KAEA (SUDPRO), a member of the ubiquitous KEOPS/EKC protein complex, regulates the arginine catabolic pathway and the expression of several other genes in Aspergillus nidulans**. Dzikowska A, Grzelak A, Gawlik J, Szewczyk E, Mrozek P, Borsuk P, Koper M, Empel J, **Szczesny P**, Piłsyk S, and others. *Gene*. 2015. 573(2):310-320.
*   **Inhibition of Porphyromonas gingivalis growth by metalloporphyrins**. Olczak T, Wojtowicz H, Maszczak-Seneczko D, Ciuraszkiewicz J, **Szczesny P**. 2010.
*   **Verification of a topology model of PorT as an integral outer-membrane protein in Porphyromonas gingivalis**. Nguyen KA, Żylicz J, **Szczesny P**, Sroka A, Hunter N, Potempa J. *Microbiology*. 2009. 155(2):328.